src/moduler/aktivitet/aktivitet-kort/AktivitetType.js

```diff
@@ -0,0 +1,38 @@
+import { FormattedMessage } from 'react-intl';
+import { Element, Undertekst } from 'nav-frontend-typografi';
+import React from 'react';
+import classNames from 'classnames';
+import PT from 'prop-types';
+import AktivitetskortEndring from './aktivitetskort-endring';
+import * as AppPT from '../../../proptypes';
+
+export const Titel = ({ aktivitet, harEndringerIAktivitet, isDragging }) =>
+    <div className="aktivitetskort__header">
+        <AktivitetskortEndring
+            harEndringerIAktivitet={harEndringerIAktivitet}
+        />
+        <Element
+            tag="h1"
+            id={`aktivitetskort__header__${aktivitet.id}`}
+            className={classNames('aktivitetskort__tittel softbreak', {
+                'aktivitetskort__tittel--drag': isDragging,
+            })}
+        >
+            {aktivitet.tittel}
+        </Element>
+    </div>;
+
+Titel.propTypes = {
+    aktivitet: AppPT.aktivitet.isRequired,
+    isDragging: PT.bool.isRequired,
+    harEndringerIAktivitet: PT.bool.isRequired,
+};
+
+export const AktivitetType = ({ type }) =>
```

Comment:
Ikke ha 2 av disse i samme fil. De burde være separert. Eslint regelen for dette er av pr nå man kommer til å bli skrudd på i fremtiden